functions/src/services/realtimeDB.ts

```diff
@@ -0,0 +1,36 @@
+import firebase from "firebase-admin";
+import { initializeApp } from "firebase/app";
+import { getDatabase } from "firebase/database";
```

Comment:
initialzeApp & getDatabase kunnen als het goed is ook met de firebase-admin gedaan worden.
dan kan die firebase import in de backend weggehaald worden.

Reply:
done